trajectorytools/export/to_pandas.py

```diff
@@ -0,0 +1,167 @@
+import collections
+import functools
+import itertools
+from typing import Dict, List, Optional
+
+import numpy as np
+import pandas as pd
+
+from variables import get_variable
+
+
+def melt_df(df: pd.DataFrame, value_name: str, var_name: str):
+    melted_df = pd.melt(
+        df,
+        id_vars=[col for col in df.columns if isinstance(col, str)],
+        value_vars=[col for col in df.columns if not isinstance(col, str)],
+        var_name=var_name,
+        value_name=value_name,
+    )
+    return melted_df
+
+
+def generate_var_df(
+    x: np.ndarray,
+    y: np.ndarray,
+    x_name: str,
+    y_name: str,
+    identities: Optional[Dict[str, List]] = None,
```

Comment:
not sure what the identities information is

Reply:
This was to pass groups of identities, but it should have been removed. I will take it out.